packages/auth/src/resourceSet/routes.ts

```diff
@@ -0,0 +1,41 @@
+import { AppContext } from '../app'
+import { BaseService } from '../shared/baseService'
+import { ResourceSetService } from './service'
+
+interface ServiceDependencies extends BaseService {
+  resourceSetService: ResourceSetService
+}
+
+export interface ResourceSetRoutes {
+  create(ctx: AppContext): Promise<void>
+}
+
+export function createResourceSetRoutes({
+  logger,
+  resourceSetService
+}: ServiceDependencies): ResourceSetRoutes {
+  const log = logger.child({
+    service: 'ResourceSetRoute'
+  })
+
+  const deps = {
+    logger: log,
+    resourceSetService
+  }
+
+  return {
+    create: (ctx: AppContext) => createResourceSet(deps, ctx)
+  }
+}
+
+async function createResourceSet(
+  deps: ServiceDependencies,
+  ctx: AppContext
+): Promise<void> {
+  const { body } = ctx.request
+  const resourceSet = await deps.resourceSetService.create(body)
+
+  ctx.body = {
+    resource_reference: resourceSet.id
```

Comment:
It sounds like we want to return `instance_id`
- https://datatracker.ietf.org/doc/html/draft-ietf-gnap-resource-servers#section-3.4
- #601

Reply:
Could this also return `introspection_endpoint` to save the RS from having to configure:
https://github.com/interledger/rafiki/blob/c2a7805500d944928befbe71449e6a7d5a16ec3f/packages/backend/src/config/app.ts#L78-L81
If so, should that only happen if `token_introspection_required` is included in the request?